src/winetricks

```diff
@@ -2546,10 +2555,10 @@ w_wine_version_in()
 # the environment variable WINETRICKS_BLACKLIST to disable it.
 w_workaround_wine_bug()
 {
-    if test "$WINE" = ""; then
-        echo "No need to work around wine bug $1 on Windows"
-        return 1
-    fi
+    case "$(winetricks_get_platform)" in
```

Comment:
$W_PLATFORM

Reply:
Presented makes sure that W_PLATFORM is exported assuming that `winetricks_get_platform` now outputs which platform is used or do you want me to use $W_PLATFORM instead?

Reply:
Use $W_PLATFORM

Reply:
Resolved on demand https://github.com/Winetricks/winetricks/pull/1338/commits/90690b501d6362c95f6645b66f3bc082b93e410c

Disclaimer: i still think that said function should be called directly instead of using variables.